src/app/component/Typography.jsx

```diff
@@ -1,45 +1,41 @@
 import React from "react";
 import { Typography } from "@material-ui/core";
+import theme from "../../theme";
 
 const H1 = ({ children, ...rest }) => (
-  <Typography variant="h1" {...rest}>
+  <Typography variant="h1" style={theme.typography.h1} {...rest}>
     {children}{" "}
   </Typography>
 );
 const H2 = ({ children, ...rest }) => (
-  <Typography variant="h2" {...rest}>
+  <Typography variant="h2" style={theme.typography.h2} {...rest}>
     {children}
   </Typography>
 );
 const H3 = ({ children, ...rest }) => (
-  <Typography variant="h3" {...rest}>
+  <Typography variant="h3" style={theme.typography.h3} {...rest}>
     {children}{" "}
   </Typography>
 );
 const H4 = ({ children, ...rest }) => (
-  <Typography variant="h4" {...rest}>
+  <Typography variant="h4" style={theme.typography.h4} {...rest}>
     {children}{" "}
   </Typography>
 );
 const H5 = ({ children, ...rest }) => (
-  <Typography variant="h5" {...rest}>
+  <Typography variant="h5" style={theme.typography.h5} {...rest}>
     {children}{" "}
   </Typography>
 );
-const H6 = ({ children, ...rest }) => (
-  <Typography variant="h6" {...rest}>
-    {children}
-  </Typography>
-);
```

Comment:
Can these functions be set up as one to avoid rewriting so much code?

Seems like we've abstracted Typography out of the components and put it all in this app. Not a blocking suggestion for this, but I think we can set up a Typography wrapper that takes the variant as a prop and continues to pass the rest back?

```javascript
const withTypography = (props) => <Typography variant={props.elementType} style={theme.typography[props.elementType]} {...rest}>{props.children}</Typography>
```

Maybe some extra thought into the naming so it makes some more semantic sense. but Seems like there is a lot of wet code here that can get optimized. If everyone else is okay for with these changes, I'd suggest we merge this, and open a new issue with this thought in mind.